http/codegen/testdata/payload_decode_functions.go

```diff
@@ -3178,11 +3178,8 @@ var PayloadBodyNestedUserDecodeCode = `// DecodeMethodBodyUserRequest returns a
 func DecodeMethodBodyUserRequest(mux goahttp.Muxer, decoder func(*http.Request) goahttp.Decoder) func(*http.Request) (interface{}, error) {
 	return func(r *http.Request) (interface{}, error) {
 		var (
-			body struct {
-				A *string
-				B *string
-			}
-			err error
+			body NestedType
```

Comment:
Wondering whether using `body *NestedType` and passing `body` instead of `&body` would be better. This would follow the convention we've used elsewhere better.

Reply:
We initialize non-pointer `body` in other places too.